storage/storage.go

```diff
@@ -101,6 +101,8 @@ type Storage interface {
 	CreateReport(r *entities.Report) error
 	UpdateReport(r *entities.Report) error
 	GetReportByFilename(filename string, userID int64) (*entities.Report, error)
+	GetRunningReportForUser(int64) (*entities.Report, error)
```

Comment:
Name the input arguments:

```suggestion
	GetRunningReportForUser(userID int64) (*entities.Report, error)
```